packages/git/src/browser/diff/git-diff-contribution.ts

```diff
@@ -68,7 +68,7 @@ export class GitDiffContribution extends AbstractViewContribution<GitDiffWidget>
                                 fromRevision
                             }
                         };
-                        if (fileStat.isDirectory) {
+                        if (fileStat && fileStat.isDirectory) {
```

Comment:
That is not correct I think. Previously we always had an existing file, now you say; if it exists and is a directory.

The `else` branch does not handle the missing resource. (I am not telling that it was flawless before, but you changed the semantics, didn't you?)

CC: @jbicker